src/main/java/entities/Post.java

```diff
@@ -65,5 +66,15 @@ public String getCategory() {
     public void setRecipe(Recipe recipe) {
         this.recipe = recipe;
     }
+
+    /*Comparator for sorting the list by the number of likes of a post*/
+    public static Comparator<Post> PostLikesComparator = (p1, p2) -> {
```

Comment:
I'm not sure if Comparator is being used in the correct way here. Shouldn't Post implement the Comparator interface? Maybe look at the answer here https://stackoverflow.com/questions/2839137/how-to-use-comparator-in-java-to-sort for an example